tests/pytests/integration/modules/test_cp.py

```diff
@@ -0,0 +1,685 @@
+import hashlib
+import logging
+import os
+import pathlib
+import signal
+import textwrap
+import time
+import uuid
+
+import psutil  # pylint: disable=3rd-party-module-not-gated
+import pytest
+import salt.utils.files
+import salt.utils.path
+import salt.utils.platform
+import salt.utils.stringutils
+from saltfactories.utils.ports import get_unused_localhost_port
+from saltfactories.utils.tempfiles import temp_file
+from tests.support.runtests import RUNTIME_VARS
+from tests.support.unit import skipIf
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.slow_test
+def test_get_file(tmp_path, salt_cli, salt_minion):
+    """
+    cp.get_file
+    """
+    tgt = str(tmp_path / "test_get_file")
+    salt_cli.run("cp.get_file", "salt://grail/scene33", tgt, minion_tgt=salt_minion.id)
+    with salt.utils.files.fopen(tgt, "r") as scene:
+        data = salt.utils.stringutils.to_unicode(scene.read())
+    assert "KNIGHT:  They're nervous, sire." in data
+    assert "bacon" not in data
```

Comment:
```suggestion
    tgt = tmp_path / "test_get_file"
    salt_cli.run("cp.get_file", "salt://grail/scene33", str(tgt), minion_tgt=salt_minion.id)
    data = tgt.read_text()
    assert "KNIGHT:  They're nervous, sire." in data
    assert "bacon" not in data
```
I think we no longer need the `fopen` calls, nor the `to_unicode` calls....